projekt_do_pracy_na_labach/fiszki_backend/Dockerfile

```diff
@@ -1,2 +1,39 @@
 FROM python:3.10
 
+WORKDIR /usr/src/app
```

Comment:
Polecam sobie WORKDIR uprościć, wystarczy do zwykłego `app/`. To jest tylko komentarz jb, nie wpływa na ocenę